importers/src/bible/osis/index.ts

```diff
@@ -0,0 +1,918 @@
+import { resolve } from 'path';
+import { readFileSync } from 'fs';
+import { BibleEngineImporter } from '../../shared/Importer.interface';
+
+import { parser, Tag } from 'sax';
+import { ParserContext, TagWithType } from './types';
+import {
+    IBibleContentSection,
+    IBibleContentGroup,
+    IBibleContentPhrase,
+    IBibleNote,
+    DocumentRoot,
+    DocumentPhrase,
+    DocumentGroup
+} from '@bible-engine/core';
+import { startsWithPunctuationChar } from '../../shared/helpers.functions';
+
+const DEBUG_OUTPUT_ENABLED = true;
+const DEBUG_LEVEL: 'verbose' | 'info' | 'warning' | 'error' = 'warning';
+const STRICT_MODE_ENABLED = true;
+
+export class OsisImporter extends BibleEngineImporter {
+    async import() {
+        let xml: string;
+        if (this.options.sourceData) xml = this.options.sourceData;
+        else {
+            const sourcePath = this.options.sourcePath || resolve(__dirname) + '/data/osis.xml';
+            xml = readFileSync(sourcePath, 'utf8');
+            // await streamToString(
+            //     createReadStream(sourcePath)
```

Comment:
remove commented out code

Reply:
added `sourceEncoding` config option to importers